lib/addons/coredns/index.ts

```diff
@@ -0,0 +1,14 @@
+import { EksManagedCoreAddons } from "../eks-managed-core-addons";
+
+/**
+ * Implementation of CoreDns EKS add-on.
+ */
+export class CoreDnsAddOn extends EksManagedCoreAddons {
```

Comment:
Sorry to nitpick, but for consistency `EksManagedCoreAddons` should be `EksManagedCoreAddOns`

Reply:
(nothing to be sorry about). Ideally, we should create an internal naming guideline document and just  *reference* it, to avoid trial and error approach.